packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -275,12 +280,12 @@ class FlutterDriver {
     // the isolate is already resumed. There could be a race with other tools,
     // such as a debugger, any of which could have resumed the isolate.
     Future<dynamic> resumeLeniently() {
-      _log.trace('Attempting to resume isolate');
+      _log('Attempting to resume isolate...');
```

Comment:
A lot of these messages would be just noise for most normal usages of flutter driver, but since we seem to be printing everything all the time anyway, then sure 🤷‍♂

Reply:
I audited them all and they seemed pretty reasonable.